module.exports[`creates an error`] = "
   Execution Error
   3452345

   Possible cause
   53656564

   2893475923847593

   Possible solution
   985u4986

   If you believe this is not an expected behaviour or that this error message should be improved, open an issue: https://github.com/dawson-org/dawson-cli/issues"

module.exports[`creates an error with default params`] = "
   Execution Error

   If you believe this is not an expected behaviour or that this error message should be improved, open an issue: https://github.com/dawson-org/dawson-cli/issues"